DatadogSessionReplay/Sources/SessionReplay.swift

```diff
@@ -69,6 +69,14 @@ public enum SessionReplay {
                 description: "Datadog SDK must be initialized before calling `SessionReplay.enable(with:)`."
             )
         }
+
+        guard core.get(feature: SessionReplayFeature.self) == nil else {
+            core.telemetry.send(telemetry: .debug(id: "1", message: "Session Replay has already been enabled", attributes: nil))
```

Comment:
/change-request The `id` is made for deduplication, you can use [this method](https://github.com/DataDog/dd-sdk-ios/blob/develop/DatadogInternal/Sources/Telemetry/Telemetry.swift#L288) instead.
```suggestion
            core.telemetry.debug("Session Replay has already been enabled")
```